scrapers/covid-19/govScrapers/getAustralia.js

```diff
@@ -0,0 +1,37 @@
+const nightmare = require('nightmare');
```

Comment:
the package is needed to use because the table does not fully load the table when using Axios.